docs/markdown/Getting Help/the-pants-community/team.md

```diff
@@ -48,6 +48,7 @@ Pants open source project has had many team members in over a decade. The curren
 | **Jacob Floyd**        | Gave the idea for skip_flake8 et al, which grew into a flagship feature: incremental adoption           |
```

Comment:
Some of these are quoted, and some are not... not sure whether you want to fix that now or in a followup though.

Reply:
It's not a mistake. The quotation marks are only on ones where the person actually said the words. In the long term I'd like to circle back to those who don't have a personal quote yet.